config/locales/event.en.yml

```diff
@@ -71,3 +75,6 @@ en:
     mark_as_further_actions_required: Case marked as SSCL managing case
     unmark_as_further_actions_required: Case marked as SSCL not managing case
     mark_as_awaiting_response_for_partial_case: Case marked as awaiting response following SSCL letter
+    require_further_action_desc: Further information requested by ICO
```

Comment:
"by the ICO"?

Reply:
hmm,  I copied this line from tickets, I don't mind to add 'the',  just not sure 😳

Reply:
I think it's needed based on my understanding but please double check with ticket writer, thanks